python_modules/dagster/dagster/core/execution/context/hook.py

```diff
@@ -32,12 +32,20 @@ class HookContext:
         log (DagsterLogManager): Centralized log dispatch from user code.
         hook_def (HookDefinition): The hook that the context object belongs to.
         solid (Solid): The solid instance associated with the hook.
+        op (Op): The op instance associated with the hook.
         step_key (str): The key for the step where this hook is being triggered.
+        required_resource_keys (Set[str]): Resources required by this hook.
         resources (Resources): Resources available in the hook context.
         solid_config (Any): The parsed config specific to this solid.
+        op_config (Any): The parsed config specific to this op.
         pipeline_name (str): The name of the pipeline where this hook is being triggered.
+        job_name (str): The name of the job where this hook is being triggered.
         run_id (str): The id of the run where this hook is being triggered.
         mode_def (ModeDefinition): The mode with which the pipeline is being run.
+        solid_exception (Optional[BaseException]): The thrown exception in a failed solid.
+        op_exception (Optional[BaseException]): The thrown exception in a failed op.
+        solid_output_values (Dict): Computed output values in a solid.
+        op_output_values (Dict): Computed output values in an op.
```

Comment:
Hmmmm I guess it doesn't hurt to have both documented at top level like this. cc @sryza I think this brings up an interesting question regarding how we update API docs as we migrate. Do we keep around old props in docs, or just move to new ones?

For now I think we still need both, since accompanying docs haven't yet been updated